content/post/getting-started-with-zfs-on-osx.md

```diff
@@ -0,0 +1,173 @@
++++
+author = "vmrob"
+date = "2015-11-07T13:16:31-07:00"
+description = "So you've heard of next gen filesystems like ZFS and BTRFS and can't help but to get a little jealous that your simple-minded, consumer filesystem doesn't have newfangled features like copy-on-write, snapshots, de-duplication, and block-level compression. Using these features, I was able to halve the storage of my virtual machines saving me over 125 GB. In this article, I'll show you how to master ZFS on OS X using nothing but your existing Apple hardware."
+tags = ['zfs', 'osx', 'workflow', 'power user']
+title = "Getting started with ZFS on OS X"
+
++++
+
+So you've heard of "next gen" filesystems like [ZFS](https://en.wikipedia.org/wiki/ZFS) and [BTRFS](https://en.wikipedia.org/wiki/Btrfs) and can't help but to get a little jealous that your simple-minded, consumer filesystem doesn't have newfangled features like [copy-on-write](https://en.wikipedia.org/wiki/Copy-on-write), snapshots, de-duplication, and block-level compression. Using these features, I was able to halve the storage of my virtual machines saving me over 125 GB. In this article, I'll show you how to master ZFS on OS X using nothing but your existing Apple hardware.
+
+# Disclaimer
+
+In my personal experience, ZFS on OS X is still somewhat buggy. I've never lost any data, but the occasional VFS deadlock isn't too uncommon and when that happens, the only thing you can really do is shut down the computer. When the VFS (Virtual Filesystem) locks up, everything that uses it, including just about all of your user-level programs, will block on vfs calls indefinitely. Your computer will appear to function but then the beach ball of death will silently take over everything.
```

Comment:
introducing an acronym is typically done in via *thing to be introduced (TTBI)*. i.e. full name first. and this should happen the first time it's used.

there's a lowercase instance of "vfs" here